src/ansible_creator/utils.py

```diff
@@ -65,153 +64,174 @@ def expand_path(path: str) -> Path:
 
 
 @dataclass
-class Copier:
-    """Configuration for the Copier class.
+class DestinationFile:
```

Comment:
I feel like this stopped being a good name halfway through making this PR. Open to ideas of a better way to refer to a source-dest file pair.

Reply:
SrcDestPair?